esp/esp/django_settings.py

```diff
@@ -290,7 +290,7 @@
 
 CONTACTFORM_EMAIL_CHOICES = (
     ('esp','Unknown'),
-    ('general','General ESP'),
+    ('general','General Program'),
     ('esp-web','Web Site Problems'),
```

Comment:
May as well change this to "website" while you're at it!

Reply:
Good catch!